src/Logging/README.md

```diff
@@ -11,6 +11,10 @@
 support requests, bug reports, or development contributions should be directed to
 that project.
 
+By default, this client uses REST over HTTP/1.1. For improved performance, or to make use of the GAPIC clients, we recommend installing the gRPC PHP extension. For installation instructions, [see here](https://cloud.google.com/php/grpc).
```

Comment:
WDYT about rewording or dropping the first sentence, since we technically could default to gRPC assuming they already have the extension installed?

`If gRPC is not already installed, this client uses REST over HTTP/1.1.`?